docs/en_us/platform_api/Makefile

```diff
@@ -1,164 +0,0 @@
-# Makefile for Sphinx documentation
```

Comment:
@andy-armstrong: See this ticket for the plan for that document: https://openedx.atlassian.net/browse/DOC-3660